addons/fleet/models/fleet_vehicle.py

```diff
@@ -281,7 +281,7 @@ def return_action_to_open(self):
         self.ensure_one()
         xml_id = self.env.context.get('xml_id')
         if xml_id:
-            res = self.env['ir.actions.act_window'].for_xml_id('fleet', xml_id)
+            res = self.env['ir.actions.act_window']._for_xml_id('fleet', xml_id)
```

Comment:
Wow: "Get some random external id from the context."

Reply:
Yeah, this is why I added the assert and filter, definitly not safe otherwise (and still not so good)